content/booklets/adacore-technologies-for-airborne-software/analysis.rst

```diff
@@ -0,0 +1,3180 @@
+.. include:: common_defs.rst
```

Comment:
We should use the common file containing definitions for all courses:

```
.. include:: ../../global.txt
```

Any definition that is missing can be added to that file.